Mechanic2.roboFontExt/lib/mechanic2/extensionItem.py

```diff
@@ -192,7 +182,7 @@ def remoteInstall(self, forcedUpdate=False, showMessages=False):
         if extensionPath:
             # if found get the bundle and install it
             bundle = ExtensionBundle(path=extensionPath)
-            bundle.install(showMessages=showMessages)
```

Comment:
I guess you didnt start from the latest version...  it seems like this commit https://github.com/robofont-mechanic/mechanic-2/commit/1901e663e1cc63c5485fa0cc05e9a423961560ee was not added to your branche

Reply:
Let me pull it in.

Reply:
I did pick up the commit (my fork is from a few days ago) but I think I broke this specific bit in the process of rewriting the rest. One of the drawbacks of my solution is that I had to resort to async calls which make the code flow really hard to follow. If nothing else, you can pick up the general idea in [urlreader.py](https://github.com/verbosus/mechanic-2/blob/master/Mechanic2.roboFontExt/lib/urlreader.py) and take it from there. Let me see if I can bring what 1901e663e1cc63c5485fa0cc05e9a423961560ee was trying to do.

Reply:
OK, I picked up the showMessages change that was originally in 1901e663e1cc63c5485fa0cc05e9a423961560ee, please give it a go now @typemytype